charts/router/templates/router-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Capabilities.APIVersions.Has (include "rbacApiVersion" .) }}
+kind: ClusterRole
+apiVersion: {{ template "rbacApiVersion" . }}
+metadata:
+  name: deis:deis-router
+  labels:
+    app: deis-router
+    heritage: deis
+rules:
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch"]
```

Comment:
The only deployment that the router is concerned with is its own. It's retrieved by name-- i.e. a get. Could you please try removing list and watch here and verify that the router still works?